tests/helpers/Auth.ts

```diff
@@ -0,0 +1,18 @@
+import { Page } from "@playwright/test";
+
+export const login = async (page: Page) => {
+  let shouldLogin = process.env.TRUSTIFY_AUTH_ENABLED;
+
+  if (shouldLogin === "true") {
+    let userName = process.env.TRUSTIFY_AUTH_USER ?? "admin";
+    let userPassword = process.env.TRUSTIFY_AUTH_PASSWORD ?? "admin";
+
+    await page.goto("/");
+
+    await page.fill('input[name="username"]:visible', userName);
+    await page.fill('input[name="password"]:visible', userPassword);
+    await page.click('input[type="submit"]:visible');
```

Comment:
I discovered that Keycloak 26 switched from "input" to "button" for the login button. Therefore 

```suggestion
    await page.keyboard.press("Enter");
```


![Screenshot From 2025-02-24 14-13-21](https://github.com/user-attachments/assets/7b05a49b-e1b8-4663-ac6d-de0fc97c20b9)